htdocs/install/mysql/migration/repair.sql

```diff
@@ -576,3 +576,4 @@ DELETE FROM llx_rights_def WHERE module = 'hrm' AND perms = 'employee';
 -- DROP TABLE tmp_bank2;
 -- CREATE TABLE tmp_bank2 SELECT b.rowid, b.amount, p.rowid as pid, p.amount as pamount, p.multicurrency_amount as pmulticurrencyamount FROM llx_bank as b INNER JOIN llx_bank_url as bu ON bu.fk_bank=b.rowid AND bu.type = 'payment_supplier' INNER JOIN llx_paiementfourn as p ON bu.url_id = p.rowid WHERE p.multicurrency_amount <> 0 AND p.multicurrency_amount <> p.amount;
 -- UPDATE llx_bank as b SET b.amount_main_currency = (SELECT tb.pamount FROM tmp_bank2 as tb WHERE tb.rowid = b.rowid) WHERE b.amount_main_currency IS NULL;
+ALTER TABLE `llx_c_tva` CHANGE `code` `code` varchar(20) AFTER `fk_pays`;
```

Comment:
In table llx_c_tva, size is 10.
To increase it, you must both  modify the sql file that create the table and the sql file of the migration.